ee/codegen/src/project.ts

```diff
@@ -8,24 +8,13 @@ import { AstNode } from "@fern-api/python-ast/core/AstNode";
 import { VellumEnvironmentUrls } from "vellum-ai";
 
 import * as codegen from "./codegen";
+
 import {
   GENERATED_DISPLAY_MODULE_NAME,
   GENERATED_NODES_MODULE_NAME,
-} from "./constants";
-import { createNodeContext, WorkflowContext } from "./context";
-import { InputVariableContext } from "./context/input-variable-context";
-import { WorkflowOutputContext } from "./context/workflow-output-context";
-import { ErrorLogFile, InitFile, Inputs, Workflow } from "./generators";
-import {
-  NodeDefinitionGenerationError,
-  ProjectSerializationError,
-} from "./generators/errors";
-import { BaseNode } from "./generators/nodes/bases";
-import { GuardrailNode } from "./generators/nodes/guardrail-node";
-import { InlineSubworkflowNode } from "./generators/nodes/inline-subworkflow-node";
-import { SearchNode } from "./generators/nodes/search-node";
-import { TemplatingNode } from "./generators/nodes/templating-node";
-
+} from "src/constants";
+import { createNodeContext, WorkflowContext } from "src/context";
+import { InputVariableContext } from "src/context/input-variable-context";
```

Comment:
vitest treats relative and absolut paths as different modules, so we should always be using absolute